README.md

```diff
@@ -47,12 +47,51 @@ You can also create your own stragery. This is as simple as creating a method on
 def self.from_params(params)
   user = User.find_by(something: params[:something])
   return nil unless user.present?
-  
+
   return nil unless user.password_valid?(params[:password])
   user
 end
 ```
 
+### Invalidate Tokens
+
+If you want to be able to invalidate tokens from the server, then you can add `Tokenable::Verifier`.
+
+```ruby
+class User < ApplicationRecord
+  include Tokenable::Verifier
+end
+```
+
+And running the following migration:
+
+```bash
+rails g migration AddTokenableVerifierToUsers tokenable_verifier:uuid
```

Comment:
`uuid` is a bit strict. We want to allow longer keys, and just alphanumeric ideally.

Plus in future we can support multiple keys by just splitting on `,` for backwards compatibility and ability to cycle. We can't do that with a `uuid` field on the db.